modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/js/SiteNavigationMenuEditor.es.js

```diff
@@ -347,7 +347,7 @@ class SiteNavigationMenuEditor extends State {
 
 		formData.append(`${this.namespace}order`, data.dragOrder);
 
-		return fetch(this.editSiteNavigationMenuItemParentURL, {
+		return Liferay.Util.fetch(this.editSiteNavigationMenuItemParentURL, {
 			body: formData,
 			credentials: 'include',
```

Comment:
1. `Liferay.Util.fetch` adds `credentials: 'include'`, so we can remove it from usages. 
1. If usage is in *.es.js, instead of adding `Liferay.Util.`, we can `import {fetch} from 'frontend-js-web'`